dask/base.py

```diff
@@ -950,6 +951,7 @@ def tokenize(*args, **kwargs):
         type(Ellipsis),
         datetime.date,
         datetime.timedelta,
+        pathlib.Path,
```

Comment:
Should this be `pathlib.PurePath`?

(Or, maybe the test needs to use `pathlib.Path`?)

ref: https://github.com/dask/dask/runs/7131483064?check_suite_focus=true#step:6:22585